pipeline-scripts/commonlib.groovy

```diff
@@ -21,6 +22,7 @@ ocp4Versions = [
 
 // Which versions should undergo merges from origin->ose
 ocpMergeVersions = [
+    "4.6",
```

Comment:
i would wait to do this until we have the builds running fairly smoothly or we'll likely be running failures every 15 minutes.
maybe this should be broken out into a separate step in the GA playbook...